resources/com/github/coreos/job.json

```diff
@@ -0,0 +1,52 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "clean-up",
+    "annotations": {
+      "openshift.io/display-name": "Clean Up Job",
+      "description": "Clean Up Job",
+      "template.openshift.io/bindable": "false"
+    }
+  },
+  "objects": [
+  {      
+  "apiVersion": "batch/v1",
```

Comment:
Missing some indentation here. Might be worth passing through `jq` to get it pretty-printed (though as mentioned in the first comment, hopefully we switch to YAML soon! :) )